apphub/image_classification/lenet_mnist/tensorflow/lenet_mnist.py

```diff
@@ -42,28 +58,38 @@ def LeNet(input_shape=(28, 28, 1), classes=10):
     return model
 
 
-def get_estimator():
```

Comment:
same as before, let's create a separate file in tensorflow folder.

Reply:
added the extra files for now